src/models/SessionEvent.ts

```diff
@@ -38,6 +39,10 @@ export class SessionEvent {
 	@IsNotEmpty()
 	userId!: string;
 
+	@IsString()
+	@IsNotEmpty()
```

Comment:
its possible that clientSessionId is not present

Reply:
Resolved https://github.com/alphagov/di-ipvreturn-api/pull/56/commits/3bef98ecba1b71c45d234cc1cd1f55168da8e20b